llama-index-integrations/llms/llama-index-llms-databricks/pyproject.toml

```diff
@@ -0,0 +1,56 @@
+[build-system]
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+# Feel free to un-skip examples, and experimental, you will just need to
+# work through many typos (--write-changes and --interactive will help)
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
+[tool.llamahub]
+contains_example = false
+import_path = "llama_index.llms.databricks"
+
+[tool.llamahub.class_authors]
+DataBricks = "busaud"
+
+[tool.mypy]
+disallow_untyped_defs = true
+# Remove venv skip when integrated with pre-commit
+exclude = ["_static", "build", "examples", "notebooks", "venv"]
+ignore_missing_imports = true
+python_version = "3.8"
+
+[tool.poetry]
+name = "llama-index-llms-databricks"
+version = "0.1.0"
+description = "llama-index llms databricks integration"
+authors = ["Abdulaziz Almuhaidib <abdulaziz.almuhaidib.97@gmail.com>"]
+license = "MIT"
+readme = "README.md"
+packages = [{include = "llama_index/"}]
+
+[tool.poetry.dependencies]
+python = ">=3.8.1,<4.0"
+llama-index-core = "^0.10.0"
```

Comment:
`llama-index-llms-openai-like` should be a dependency

Reply:
Added.